packages/manager/src/queries/cloudpulse/metrics.ts

```diff
@@ -18,7 +18,7 @@ export const useCloudPulseMetricsQuery = (
   serviceType: string,
   request: CloudPulseMetricsRequest,
   obj: {
-    authToken: string;
+    authToken?: string;
```

Comment:
@ankitaakamai , did we need this change, if we keep the previous data true, since we now drive the logic based isJweTokenFetching